path_planning_plugins/include/path_planning_plugins.h

```diff
@@ -0,0 +1,50 @@
+#ifndef PATH_PLANNING_PLUGINS_H
+#define PATH_PLANNING_PLUGINS_H
+
+#include "path_planning_base.h"
+#include <godel_process_path_generation/utils.h>
+#include <godel_process_path_generation/polygon_utils.h>
+#include <godel_process_path_generation/polygon_pts.hpp>
+#include "profilometer_scan.h"
+#include "mesh_importer.h"
+
+namespace path_planning_plugins
+{
+
+namespace Openveronoi
+{
+  bool populatePlanningParameters(ros::NodeHandle& nh, path_planning_base::PlanningParams& params);
```

Comment:
I don't anticipate a user would call it, but it's common to both ScanPlanner and BlendPlanner, and I wanted to avoid copy/paste. I could write it as a default implementation within path_planner_base and have both inherit it, or make it a private function in each subclass, but having it exist within the Openveronoi namespace seemed like a good compromise.